packages/builders/src/components/textInput/Assertions.ts

```diff
@@ -0,0 +1,17 @@
+import { TextInputStyle } from 'discord-api-types/v9';
+import { z } from 'zod';
+import { customIdValidator } from '../Assertions';
+
+export const textInputStyleValidator = z.nativeEnum(TextInputStyle);
+export const minLengthValidator = z.number().min(0).max(4000);
```

Comment:
```suggestion
export const minLengthValidator = z.number().int().min(0).max(4000);
```

Reply:
Space's request to add `.int()` to length validators wasn't applied